docs/reference/config.md

```diff
@@ -438,7 +438,7 @@ The following settings are available:
 `azure.batch.terminateJobsOnCompletion`
 : :::{versionadded} 23.05.0-edge
   :::
-: When the workflow completes, set all jobs to terminate on task completion (default: `true`).
+: Set all jobs to terminate on task completion (default: `true`).
```

Comment:
```suggestion
: If `true`, all jobs terminate on task completion (default: `true`).
```
Very minor suggestion. TLDR; we are working towards standardizing language for configuration options. Please check meaning is retained.